recipes/resiprocate/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+import shutil
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+
+class ResiprocateConan(ConanFile):
```

Comment:
https://cpp.libhunt.com/conan-changelog/1.29.1

https://github.com/conan-io/conan-center-index/pull/3432/files#diff-fadd84d6f3b955981505cfa3e8a382f561f228388350c42cdd88a3d63b3e100fR73

```suggestion


required_conan_version = ">=1.29.1"

class ResiprocateConan(ConanFile):
```